docs/0.3/purchase_order/test_po.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this doc be under community/ (like the link to it?)

Reply:
no the link was wrong. It's fixed now